src/net/sf/memoranda/ui/EventsPanel.java

```diff
@@ -579,7 +613,9 @@ static void removeEventB_actionPerformed(ActionEvent e, Event ev) {
      * Description: Performs actions when the day radio button is selected. This is the default 
      * view as normal - meaning editing can be performed. US-53.
      */
-    void dayRbEventB_actionPerformed(ActionEvent e) {
+    private void dayRbEventB_actionPerformed(ActionEvent e) {
```

Comment:
Since you didn't write these methods, I don't think it is of much concern, but if you decided to these methods should start with an underscore since they are private and contain no underscores in the name. I changed mine to look something like this: "_dayRbEventActionPerformed"